client/lib/feed-post-store/constants.js

```diff
@@ -1,16 +1,8 @@
 /** @format */
 
-/**
- * External dependencies
- */
-
-import keyMirror from 'key-mirror';
-
-export default {
-	action: keyMirror( {
-		FETCH_FEED_POST: null,
-		RECEIVE_FEED_POST: null,
-		MARK_FEED_POST_SEEN: null,
-		RECEIVE_NORMALIZED_FEED_POST: null,
-	} ),
+export const action = {
```

Comment:
could we move to named exports here for each constant, rather than exporting an object? Hopefully this whole store goes away soon, so I'm not going to hold things up for it..

Reply:
that's a question up for grabs in all of these. in some cases I figured that the importing module was using enough of the action object that splitting it apart didn't make much sense.